src/hooks/protections/received_event.cpp

```diff
@@ -867,7 +881,15 @@ namespace big
 				return;
 			}
 
-			scan_explosion_event(source_player, buffer);
+			scan_explosion_event(plyr, source_player, buffer);
+
+			// dont accept event in died state

```

Comment:
Don't let the game handle the event if we're dead you mean? We're still accepting the event (from the attackers POV).

Reply:
if we are dead the explosion event is useless to us, it will only create explosions around the dead camera scene

Reply:
hmn i think it will create problem since it is not specific

Reply:
I was more targeting the wording used in the comment, we're really still accepting the event just not letting the game process it and create the explosion on our end.

Reply:
my main objective was to remove frame drop due to explosion ptfx effect when we are in died state and someone is spamming explosions.

Reply:
right we are just sending the acknowledgement but in game we are not processing the explosion. I have not tested this with scenarios like when you die and someone else die in meantime bcz of explosion or someone's car get exploded. Therefore i removed it.